specification/dimensions/usagetype.md

```diff
@@ -0,0 +1,42 @@
+# Usage Type
```

Comment:
Strongly oppose this being called UsageType. Pricing model makes sense other than the fact that we have put a sub-property of 'unused commitment' as a first-level value for this column IMO. 

I would use a value like 'Committed' or 'Committed Usage' and then have a separate property to say its used vs unused. If we're against creating an explicit column, maybe this can be figured out based on the fact that the resource id is a commitment OR the resource type for the line is a commitment discount (as opposed to some other actual resource you would use - e.g. lambda, vm, etc.) to know that the purchased discount didn't apply to actual resources.

Reply:
I would propose ChargeSubType:
- The 'Sub' pattern already exists with SubAccountId/Name
- The ChargeSubType dimension can be used to subcategorize other types of charges
- If find the pattern ChargeType/ChargeSubType simple to handle in visual reporting (for example PowerBI) for both filtering and presentation. For example, I love the columns the MeterCategory and MeterSubCategory in Azure billing data for this.

I would propose the same pattern for Service: ServiceName, SubServiceName.

Reply:
FYI - After some discussion, we decided to split this column into 2: `PricingModel` and `CommitmentUtilization` (name TBD), where this column will cover `Used` and `Not Used` capacity, spend, and usage commitments.

Reply:
There might be a risk on dispatching the information over different columns for both user understanding and data filtering & presentation. I do not know the rationale of the split, but I actually miss the holistic view / while picture on how charges and commitments are organized / structured.